client/a8c-for-agencies/sections/marketplace/controller.tsx

```diff
@@ -1,23 +1,35 @@
 import { type Callback } from '@automattic/calypso-router';
+import page from '@automattic/calypso-router';
 import getSites from 'calypso/state/selectors/get-sites';
-import MainSidebar from '../../components/sidebar-menu/main';
+import MarketplaceSidebar from '../../components/sidebar-menu/marketplace';
 import AssignLicense from './assign-license';
+import HostingOverview from './hosting-overview';
 import IssueLicense from './issue-license';
 
-export const marketplaceContext: Callback = ( context, next ) => {
+export const marketplaceContext: Callback = () => {
+	page.redirect( '/purchases/licenses' );
```

Comment:
Sorry, just realised this should be `/marketplace/products`?

Reply:
Good catch. This has been fixed 👍